ovs-with-p4-arch.rst

```diff
@@ -1,20 +1,21 @@
-Introduction
--------------------------------------------------------------------------------
+Introduction:
 -------------------------------------------------------------------------------
```

Comment:
I wonder if we should shorten these to match the text above.

Reply:
I added the colon on purpose